### PR TITLE
fix: remove useless GC

### PR DESCRIPTION
**Pull request by gdalle**
- Comment out `GC.gc()` (was taking a lot of time)
- Add `src/results` to `.gitignore`
- Fix `set_silent(m)` (because `m` was not defined)
- Fix and generalize `UWriteResults` to allow the empty `String`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,4 @@
 1-R-RSPSysimage.so
 src/debug/
 ext/results/
-
+src/results/
```
Comment (jkhamphousone) [inline .gitignore]: You forgot the empty line :)

**Review by jkhamphousone — CHANGES_REQUESTED** [review]
This is awesome, I am grateful! :)

File: .gitignore (hunk 1 above)
Comment (gdalle) [reply .gitignore]: done!

**Review by jkhamphousone — APPROVED** [review]
Perfect! This is super :)

**Pull request merged**